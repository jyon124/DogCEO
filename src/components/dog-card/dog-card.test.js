import React from 'react';
import DogCard from './dog-card.js';

import Enzyme, { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() })

describe('DogCard', () => {
    it('renders a DogCard component', () => {
        const testData = "testImage.jpeg";
        const wrapper = shallow(<DogCard dog={testData} key={testData+Math.random()} />)
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});