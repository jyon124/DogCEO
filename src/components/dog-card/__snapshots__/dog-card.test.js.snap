// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DogCard renders a DogCard component 1`] = `
<Fragment>
  <div
    className="dog-img-wrapper"
  >
    <img
      alt="dog img"
      className="dog-img"
      src="testImage.jpeg"
    />
  </div>
</Fragment>
`;
